content/reference/admin/private_locations/configuration/kubernetes/index.md

```diff
@@ -43,11 +43,13 @@ control-plane {
       # Namespace (optional, default based on kubernetes configuration)
       namespace = "gatling"
       # Certified image configuration
+      # They are hosted on Docker Hub, and available for the linux/amd64 and linux/arm64 platforms
       image {
         type = certified
         java = latest # Possible values : 8, 11, 17 or latest
       }
       # Custom image configuration
+      # You can build your own images from https://github.com/gatling/frontline-injector-docker-image
```

Comment:
Copied from the rather terse explanation for custom Kubernetes load generators on self-hosted... We may want to add more explanations?

Reply:
not now